hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/HBCKServerCrashProcedure.java

```diff
@@ -169,4 +170,16 @@ public boolean visit(Result result) throws IOException {
       return this.reassigns;
     }
   }
+
+  /**
+   * The RegionStateNode will not have a location if a confirm of an OPEN fails. On fail,
+   * the RegionStateNode regionLocation is set to null. This is 'looser' than the test done
+   * in the superclass. The HBCKSCP has been scheduled by an operator via hbck2 probably at the
+   * behest of a report of an 'Unknown Server' in the 'HBCK Report'. Let the operators operation
+   * succeed even in case where the region location in the RegionStateNode is null.
+   */
+  @Override
+  protected boolean isMatchingRegionLocation(RegionStateNode rsn) {
```

Comment:
I'm good w/ this. Just need to watch it.

I've added changes and then found I sometimes made the situation worse. Need to watch out for that here but it is HBCKSCP, run by the operator, so they should be looking at the result when manually running hbck2.

Reply:
Is it that the RSN is null but hbase:meta is an 'unknown server'?

Reply:
Yeah... it seems but the compare to RSN fails so we don't clean the unknown server... ok. Got it.

Reply:
Yes, this is my understanding. Thanks for the review!